datasrc/network.py

```diff
@@ -120,6 +120,9 @@
 		NetIntAny("m_FromY"),
 
 		NetTick("m_StartTick"),
+        
+		NetIntAny("m_OuterColor"),
+		NetIntAny("m_InnerColor"),
```

Comment:
This doesn't look like it contains backward compatibility.

Perhaps make a new `NetObject` out of it?

Reply:
I don't like the idea of two netobjects describing one laser. Seems complex.

Reply:
> I don't like the idea of two netobjects describing one laser. Seems complex.

Do you have a suggestion how to solve the backward compatibility?

Reply:
I can't think of anything clean to keep backward compatibility. So I would suggest breaking it.

Reply:
I agree with breaking it as well

Reply:
That would mean the merge has to wait for 0.8

Reply:
Uh,  I thought it would be easier & faster to break it lol
I can try to make it backwards compatible.